### DISCUSSION TITLE
The tab_new function doesn't work after the chrome version is updated.

### DISCUSSION THREAD

**Discussion by kim-chung-man** [ultrafunkamsterdam/undetected-chromedriver#1113] [General]
Chrome's version is: 111.0.5563.65
undetected_chromedriver version is : 3.1.5r4

I would like some help on how to resolve this issue.

**Comment by dempc6469** [comment]
workaround here ... though would be particurlary helpful if the tab_new worked like before

https://github.com/ultrafunkamsterdam/undetected-chromedriver/issues/380

**Comment by jjjchens235** [comment]
> workaround here ... though would be particurlary helpful if the tab_new worked like before

The solution in your link is `tab_new`, which is currently not working, am I missing something?

**Comment by dempc6469** [comment]
For me, and YMMV, I'm able to do a  `driver.execute_script(f"window.open('<url>');")` and then as long as I don't close out of the welcome page/tab for chrome I'm able to successfully open as many new tabs as I need using `driver.execute_script(f"window.open('<url>');")`. Keeping the initial chrome welcome tab open was necessary for me. If I closed that tab and worked off the  `driver.execute_script(f"window.open('<url>');")` initial tab, popup blocker would catch the new tabs and suppress them. I might add that I"m running browser version 111 but specifying `version_main=110` when i create a new instace of uc_driver. Good luck!

**Reply by dempc6469** [reply]
after looking over my code, it appears, even for my first tab I load I user the java cod to open the first window rather than driver.get() or driver.tab_new()

Just wanted to clarify for anyone experiencing the same. I'll amend the last post

**Comment by kim-chung-man** [comment]
Thanks to everyone who answered
driver.execute_script(f"window.open('<url>');")
It blocks pop-ups, but it works
But I still don't know why driver.tab_new(url) doesn't work.
